INSTALL.md

```diff
@@ -214,24 +197,30 @@ yarn install
 
 ### Prerequisites
 
-#### Ruby version
-
-The project supports **2.6.x**.
+#### Ruby and Node.js
 
-Verify you have a proper version by running on your terminal:
+The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 12](https://nodejs.org/en/download/)**. Verify you have a proper version by running on your terminal:
 
 ```bash
-ruby -v
+ruby -v && node -v
 ```
 
-> You can use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install ruby.
+[asdf](https://asdf-vm.com/guide/getting-started.html#global) is a convenient tool version manager that reads
+from the `.tool-versions` file included in this project. This way you won't have to worry about the versioning
+of these packages and it will override the system's ruby version. After installing the required dependencies
+and asdf you need to install a plugin for each individual package:
 
-#### Node version
+```sh
+apt install curl git
+git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.10.2
 
-The project supports **Version: 12.X.X**.
+asdf plugin add ruby https://github.com/asdf-vm/asdf-ruby.git
+asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+```
 
-```bash
-dnf module install nodejs:12
```

Comment:
I prefer to keep info about native way to install nodejs as well. We should not enforce one to use asdf. We can make it easy though.